frost-core/src/signature.rs

```diff
@@ -34,8 +34,9 @@ where
 
         let R_serialization = &R_bytes.try_into().map_err(|_| Error::MalformedSignature)?;
 
+        let one = <<C::Group as Group>::Field as Field>::one();
```

Comment:
Why `one`, and not `zero()`? That's basically what `Serialization::default()` did before

Reply:
Good point, no idea why I picked `one()`. Changed to  `zero()` in 17c80c1

Reply:
Nice 😎